### ISSUE TITLE
Accessibility feedback - Midpoint review - STEM Automated Denial

### ISSUE THREAD

**Issue by noahgelman** [department-of-veterans-affairs/va.gov-team#21058]
Review and provide accessibility feedback for https://bahdigital.invisionapp.com/share/G7IAG7A2XKD#/screens

## Thoughts/questions
- No accessibility issues are observed

## Feedback
- **Must** = the feedback must be applied
- **Should** = the feedback is best practice
- **Consider** = suggestions/enhancements

**Must:**
- Form download links should include the title of the form in the link alt/aria-label text so users who are tabbing through with a screen reader have context of what the forms are.

**Should:**
- Make sure Claim Status circle icon passes color contrast. This will be a consideration for WCAG2.1 [SC 1.4.11: Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html).
- Link text should be underlined in the body blocks

**Consider:**
- Maybe elaborate on what to do with the downloaded forms (where to submit)

**Comment by 1Copenut** [comment]
I added a couple of items to Noah's comments, and agree 💯 with his feedback.

**Comment by amyknox** [comment]
@JenLJ @watersemma - fysa.  Let's review Monday

**Comment by amyknox** [comment]
Thank you Noah & Trevor.  All Must & Should input looks good. We're going to pass on "Consider" because the page is too long already, despite our efforts to trim it.

**Comment by amyknox** [comment]
@noahgelman @1Copenut - want to confirm the aria label styling syntax for the PDF download links: 

Linked text:  Download VA Form 20-0995
aria label: `Download Decision Review Request for Supplemental Claims VA Form 20-0995`

**Comment by noahgelman** [comment]
@amyknox Looks good
